styles.css

```diff
@@ -1,539 +1,179 @@
-html-css/anastasiia
-:root {
-  --light-yellow: #f5e8b9b2;
-  --purpple: #b597f4;
-  --purple2: #ab87f6;
-  --pink: #f7aec7;
-  --white: #fff;
-  --blue: #c1eafc;
-  --dark_blue: #77cbef;
-}
-@font-face {
-  font-family: "Sniglet";
-  src: url(Sniglet/Sniglet-Regular.ttf);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
-
 body {
-  font-family: "Sniglet", sans-serif;
-  line-height: 1.5;
+  font-family: Arial, sans-serif;
+  background-color: #f4f4f4;
+  color: #2b1d1d;
+  line-height: 1.6;
 }
 
-/*Navigation*/
-
-.navigation {
-  background: var(--pink);
-  padding: 10px 10px;
-  display: flex;
-  justify-content: flex-end;
-  align-items: center;
-  margin-bottom: 5px;
-  width: 100%;
-  height: 50px;
-  font-size: 18px;
-  position: fixed;
+header {
+  background-color: #333;
+  color: #fff;
+  text-align: center;
+  padding: 1rem 0;
 }
 
-.nav-list {
-  list-style: none;
-  padding: 0;
-  display: flex;
-}
-.nav-list a {
-  margin: 50px;
-  text-decoration: none;
-  color: var(--white);
+header {
+  background-image: url("https://www.nouri.co/images/Historical_Photo/nasir-al-mulk-mosque.jpg");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  color: white;
+  text-align: center;
 }
 
-.navigation ul li a:hover {
-  color: var(--dark_blue);
-  font-size: 26px;
+#profile-image {
+  width: 200px;
+  height: 200px;
+  border-radius: 50%;
+  display: block;
+  margin: 0 auto;
+  border: 5px solid #fff;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+  object-fit: cover;
 }
-/*Section 1
-welcome-section */
 
-#welcome-section {
-  display: flex;
-  flex: 1;
-  height: 100vh;
-  justify-content: center;
-  align-items: center;
+#welcome-text {
```

Comment:
I would recommend to change color or move this p tag somewhere else, so it has better visibility 😃